.travis.yml

```diff
@@ -1,24 +1,34 @@
-sudo: false
 dist: xenial
 language: python
-matrix:
-  fast_finish: true
 cache:
   directories:
   - "$HOME/.cache/pip"
 
-python:
+os: linux
+_python:
   - "3.6"
   - "3.7"
-env:
+  - "3.8"
+_env:
   - NUMPY_VERSION=1.17
   - NUMPY_VERSION=1.16
   - NUMPY_VERSION=1.15
 
 jobs:
+  fast_finish: true
   include:
     - python: "2.7"
       env: NUMPY_VERSION=1.16
+    - python: "3.8"
+      env: NUMPY_VERSION=default
+      install:
+          - travis_retry pip install --upgrade pip
+          - travis_retry pip install --upgrade setuptools
+          - travis_retry pip install pytest pytest-cov coveralls setuptools-scm
+          - travis_retry pip install h5py
+          - pip install -vv .
+          - python -c 'import emcee;print(emcee.__version__)'
+          - travis_retry python setup.py develop
```

Comment:
Instead of running `setup.py develop` here, it would be better to have `pip install -e -vv .` above, I think!